packages/web/src/app/doc-pages/about/tutorial/angular-tutorial/angular-tutorial.component.html

```diff
@@ -0,0 +1,115 @@
+<!--BASIC TUTORIAL-->
+<h3 class="e-title-small e-mt-40 e-mb-0">Set up the basics</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install CSS library</div>
```

Comment:
I would suggesting using a heading-element or a paragraph-element for the titles instead of divs